client/src/app/models/site.coffee

```diff
@@ -0,0 +1,10 @@
+Backbone = require('backbone')
+
+module.exports = Backbone.Model.extend(
+  urlRoot: 'http://localhost:8989/sites'
```

Comment:
@begizi @cpjolicoeur I was able to set up a proxy using modRewrite and specifying that as middleware for gulp connect.